plugins/com.google.cloud.tools.eclipse.appengine.deploy/src/com/google/cloud/tools/eclipse/appengine/deploy/flex/FlexStagingDelegate.java

```diff
@@ -36,6 +39,7 @@ public FlexStagingDelegate(IPath appEngineDirectory) {
     this.appEngineDirectory = appEngineDirectory;
   }
 
+  // todo why is the cloudSdk argument not used here?
```

Comment:
Reason: the standard staging shells out to Cloud SDK whereas appengine-plugins-core does all copying directly via Java code. So, the interface being overridden is like a least common super set for both std and flex, which ends up having the Cloud SDK arg. There may be a way to improve it, but I don't have an immediate idea now.